lxd/storage_volumes.go

```diff
@@ -1067,9 +1143,12 @@ func storagePoolVolumePost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
-	resp = forwardedResponseIfVolumeIsRemote(s, r, srcPoolName, projectName, volumeName, volumeType)
-	if resp != nil {
-		return resp
+	// If source is set, there's no need to forward the request to a specific cluster member.
```

Comment:
Please can you include the why as well as the what, so something like

> // If source is set, there's no need to forward the request to a specific cluster member, because ...